UI/MapWnd.cpp

```diff
@@ -3931,6 +3938,12 @@ void MapWnd::RemoveProjectedFleetMovementLine(int fleet_id) {
         m_projected_fleet_lines.erase(it);
 }
 
+void MapWnd::RemoveFleetMovementLine(int fleet_id) {
+    std::map<int, MovementLineData>::iterator it = m_fleet_lines.find(fleet_id);
```

Comment:
can't you just do
````
    m_fleet_lines.erase(fleet_id);
````
?
don't really need a separate function for that, if so..,